src/codegen/operator/order_by_translator.cpp

```diff
@@ -124,9 +124,9 @@ OrderByTranslator::OrderByTranslator(const planner::OrderByPlan &plan,
                                      CompilationContext &context,
                                      Pipeline &pipeline)
     : OperatorTranslator(plan, context, pipeline),
-      child_pipeline_(this, Pipeline::Parallelism::Serial) {
-  // Aggregations happen serially (for now ...)
-  pipeline.SetSerial();
+      child_pipeline_(this, Pipeline::Parallelism::Flexible) {
```

Comment:
Do I understand correctly: The child pipeline of the Orderby (before the hashing) can be parallel, but the order-by itself has to be serial because of the aggregations.

Reply:
This is saying is that the consumer portion of the order-by (i.e., the child pipeline) can be executed in parallel, and that the production portion (i.e., the scanning of the sorter) will be serial. It has no information about other operators in the pipeline.